tests/functional/spec/features/validation/sum/value_source.spec.js

```diff
@@ -0,0 +1,156 @@
+  $(BreakdownAnswerPage.breakdown2()).setValue(breakdown2);
+  $(BreakdownAnswerPage.breakdown3()).setValue(breakdown3);
+  $(BreakdownAnswerPage.breakdown4()).setValue(breakdown4);
+  $(BreakdownAnswerPage.submit()).click();
+};
+
+const answerAndSubmitSecondBreakdownQuestion = (breakdown1, breakdown2, breakdown3, breakdown4) => {
+  $(SecondBreakdownAnswerPage.secondBreakdown1()).setValue(breakdown1);
+  $(SecondBreakdownAnswerPage.secondBreakdown2()).setValue(breakdown2);
+  $(SecondBreakdownAnswerPage.secondBreakdown3()).setValue(breakdown3);
+  $(SecondBreakdownAnswerPage.secondBreakdown4()).setValue(breakdown4);
+  $(SecondBreakdownAnswerPage.submit()).click();
+};
+
+describe("Feature: Sum of grouped answers equal to validation against answer value source ", () => {
+  beforeEach(() => {
+    browser.openQuestionnaire("test_validation_sum_against_value_source.json");
+  });
+
+  describe("Given I start a grouped answer validation survey and enter 12 into the total", () => {
+    it("When I continue and enter 3 in each breakdown field, Then I should be able to get to the total playback page", () => {
+      $(TotalAnswerPage.total()).setValue("12");
+      $(TotalAnswerPage.submit()).click();
+
+      answerAndSubmitBreakdownQuestion("3", "3", "3", "3");
+
+      expect(browser.getUrl()).to.contain(TotalPlaybackPage.pageName);
+    });
+  });
+
+  describe("Given I start a grouped answer validation survey and enter 12 into the total", () => {
+    it("When I continue to second breakdown and enter values equal to calculated summary total, Then I should be able to get to the summary page", () => {
+      $(TotalAnswerPage.total()).setValue("12");
+      $(TotalAnswerPage.submit()).click();
+
+      answerAndSubmitBreakdownQuestion("3", "3", "3", "3");
+
+      $(TotalPlaybackPage.submit()).click();
+
+      answerAndSubmitSecondBreakdownQuestion("2", "2", "1", "1");
+
+      expect(browser.getUrl()).to.contain(SubmitPage.pageName);
+    });
+  });
+
+  describe("Given I completed both grouped answer validation questions and I am on the summary", () => {
+    it("When I go back from the summary and change the total, Then I must reconfirm both breakdown questions with valid answers before I can get to the summary", () => {
+      $(TotalAnswerPage.total()).setValue("12");
+      $(TotalAnswerPage.submit()).click();
+
+      answerAndSubmitBreakdownQuestion("3", "3", "3", "3");
+
+      $(TotalPlaybackPage.submit()).click();
+
+      answerAndSubmitSecondBreakdownQuestion("2", "2", "1", "1");
+
+      $(SubmitPage.totalAnswerEdit()).click();
+      $(TotalAnswerPage.total()).setValue("15");
+      $(TotalAnswerPage.submit()).click();
+
+      answerAndSubmitBreakdownQuestion("6", "3", "3", "3");
+
+      $(TotalPlaybackPage.submit()).click();
+
+      answerAndSubmitSecondBreakdownQuestion("3", "3", "2", "1");
+
+      expect(browser.getUrl()).to.contain(SubmitPage.pageName);
+    });
+  });
```

Comment:
This test doesn't ensure you have to reconfirm with valid answers. It just shows that you walk back through it, which can change when we implement skipping completed questions. Like the existing test, I would show you submitting without a change which shows the error message meaning you have to reconfirm with correct answers. You have done it for the test case below.

Reply:
> Like the existing test, I would show you submitting without a change which shows the error message meaning you have to reconfirm with correct answers. You have done it for the test case below.

Reply:
I've added error check when submitting without a change: `expect($(BreakdownAnswerPage.singleErrorLink()).isDisplayed()).to.be.true;` but will also add another one to check it's contents:
`expect($(BreakdownAnswerPage.errorNumber(1)).getText()).to.contain("Enter answers that add up to 15");`